hazelcast/src/main/java/com/hazelcast/client/impl/protocol/codec/ClientAuthenticationCodec.java

```diff
@@ -188,18 +188,18 @@ public static class ResponseParameters {
         public boolean failoverSupported;
 
         /**
-         * Returns the list of Alto ports or null if Alto is disabled.
+         * Returns the list of TPC ports or null if TPC is disabled.
          */
-        public @Nullable java.util.List<java.lang.Integer> altoPorts;
+        public @Nullable java.util.List<java.lang.Integer> tpcPorts;
 
         /**
-         * True if the altoPorts is received from the member, false otherwise.
-         * If this is false, altoPorts has the default value for its type.
+         * True if the tpcPorts is received from the member, false otherwise.
+         * If this is false, tpcPorts has the default value for its type.
          */
-        public boolean isAltoPortsExists;
+        public boolean doTpcPortsExists;
```

Comment:
Did you generate this code with the protocol code generator?

Reply:
No. I fixed the name. isAltoPortsExists isn't proper english.

Reply:
I'll revert it. I guess the code generator generated this.

Reply:
Yeah, I am fine with this change, but it needs to happen through https://github.com/hazelcast/hazelcast-client-protocol